tensorflow/python/framework/ops.py

```diff
@@ -880,7 +880,7 @@ def _copy(self, ctx=None, device_name=None):
       self_device = self.device
 
       def grad_fun(dresult):
-        return [dresult._copy(device_name=self_device)]
+        return [dresult._copy(device_name=self_device)] if hasattr(dresult, '_copy') else [dresult]
```

Comment:
The linter will fail on this line of code.

Also, you should probably put the whole thing inside the [], like `return [dresult._copy() if hasattr(dresult, '_copy') else dresult]` which will make it easier to split into lines to fit in 80 columns.